### PR TITLE
Add support for monospace fonts

### LINKED ISSUE #14: Monospace support
Things to be added:

- [ ] Monospace?

### PR DESCRIPTION
**Pull request by dbrgn**
Add support for monospace fonts as discussed in #14. Fixes #14.

~Depends on #39 to be merged first.~

### CODE REVIEW

File: SPECIFICATION.md [hunk 1]
```diff
@@ -253,13 +254,31 @@ three different options which allow to control letter spacing:
...
 1. Trailing space of first letter (`~` space definition)
 2. Global letter spacing (`letter_spacing` from header)
 3. Leading space (leftmost X-coordinate) of second letter
 
+### 4.1.2. Monospace Fonts
+
+If a font specifies the `monospace_width` header (see subsection "Standardized
+Headers"), then that font SHOULD be layouted as a monospace font. The value of
+the `monospace_width` header corresponds to the width of every glyph's bounding
+box, excluding global spacing.
```
Comment (ubruhin) [inline SPECIFICATION.md:271]: Maybe "excluding global spacing" is not clear enough that the global `letter_spacing` is applied after every glyph?

File: SPECIFICATION.md [hunk 2]
```diff
@@ -312,6 +331,7 @@ TODO
 | license | The SPDX identifier for the license of this font. Create multiple `license` entries if the font is published under multiple licenses. | 1-n | `Apache-2.0` |
 | letter_spacing | Global letter space width for every glyph. Same unit as for polylines. If not specified, 0 is assumed (no space). | 0-1 | `1.8` (= 20% of font height) |
 | line_spacing | Vertical baseline distance of multiline texts. Same unit as for polylines. If not specified, 9 is assumed (no space). | 0-1 | `16` (= 178% of font height) |
+| monospace_width | Specifying this header marks the font as a monospace font. The value specifies the width of all glyphs (excluding global spacing). | 0-1 | `9` (= 100% of font height) |
```
Comment (ubruhin) [inline SPECIFICATION.md]: I think 9 is too wide for a typical monospace font. Probably 6 or 7 would be more realistic.

Comment (dbrgn) [reply SPECIFICATION.md]: That's just an example, right? But if you want I can adjust it to 6.

Comment (ubruhin) [reply SPECIFICATION.md]: Yes it's just an example, but the example values from the other keys are realistic for a typical font, so I'd like to see here also a realistic value.

**Comment by dbrgn** [comment]
@ubruhin fixed!

File: SPECIFICATION.md [hunk 3]
```diff
@@ -312,6 +333,7 @@ TODO
 | license | The SPDX identifier for the license of this font. Create multiple `license` entries if the font is published under multiple licenses. | 1-n | `Apache-2.0` |
 | letter_spacing | Global letter space width for every glyph. Same unit as for polylines. If not specified, 0 is assumed (no space). | 0-1 | `1.8` (= 20% of font height) |
 | line_spacing | Vertical baseline distance of multiline texts. Same unit as for polylines. If not specified, 9 is assumed (no space). | 0-1 | `16` (= 178% of font height) |
+| monospace_width | Specifying this header marks the font as a monospace font. The value specifies the width of all glyphs (excluding global spacing). | 0-1 | `6` (= 66% of font height) |
```
Comment (ubruhin) [inline SPECIFICATION.md]: With proper rounding, it's actually 67% :)

Comment (dbrgn) [reply SPECIFICATION.md]: Pfff :)

**Comment by dbrgn** [comment]
Fixed.

**Review by ubruhin — APPROVED** [review]

**Pull request merged**